cdc/server.go

```diff
@@ -392,15 +392,30 @@ func (s *Server) setUpDataDir(ctx context.Context) error {
 		return nil
 	}
 
+	cli := s.etcdClient
```

Comment:
IMHO, it is fine for a quick fix, but could u add some comments explain why the fix is needed, and ut to cover the case in the issue, thx